src/views/photos_week_loop/style.scss

```diff
@@ -78,4 +78,16 @@
   gap: 8px;
   margin-top: var(--wp--preset--spacing--20);
   text-decoration: none;
+
+  &,
```

Comment:
lol, I couldn't tell you how many times I've remembered and forgotten this to apply to both default and visited.